pwiz_tools/Skyline/Executables/SkylineBatch/SkylineBatch/Properties/Settings.cs

```diff
@@ -0,0 +1,133 @@
+﻿/*
+ * Original author: Ali Marsh <alimarsh .at. uw.edu>,
+ *                  MacCoss Lab, Department of Genome Sciences, UW
+ * Copyright 2020 University of Washington - Seattle, WA
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Configuration;
+using System.Xml;
+using System.Xml.Schema;
+using System.Xml.Serialization;
+
+namespace SkylineBatch.Properties
+{
+    internal sealed partial class Settings
+    {
+
+        [UserScopedSetting]
+        public ConfigList ConfigList
+        {
+            get
+            {
+                var list = (ConfigList) this["ConfigList"]; // Not L10N
+                if (list == null)
+                {
+                    list = new ConfigList();
+                    ConfigList = list;
+                }
+                return list;
+            }
+            set {
+                this["ConfigList"] = value; // Not L10N
+            }
+        }
+
+        [UserScopedSetting]
+        [DefaultSettingValue("false")]
+        public bool MinimizeToSystemTray {

```

Comment:
You might find it easier to define simple properties like this in "Settings.settings" using Design View in Visual Studio. You only need to add properties to "Settings.cs" if they require special code for reading or writing the value.